### ISSUE TITLE
Add a script to create these entries

### ISSUE THREAD

**Issue by krafts** [hypergig/cafe-devops#2]
Date should
1. Date should be iso-8601
2. Equipment should be constants as these will barely change
3. And most other things should be templated with a wrapper shell script so only coffee beans are changing
4. Use longname options please

**Comment by hypergig** [comment]
Yup, as I said in #4 I want to wait a little for more data and it to stabilize before investing in tooling as things tend to change a great deal in the beginning.  But yes, the goal would be an interface that checks in our data. Maybe even moving away from git as our database...
